component/provider.jsonnet

```diff
@@ -178,6 +178,11 @@ local controllerConfigRef(config) =
           resources: [ 'schedules' ],
           verbs: [ 'get', 'list', 'watch', 'update', 'patch', 'create', 'delete' ],
         },
+        {
```

Comment:
I'm a bit confused why you add this rule to the provider-kubernetes? We directly apply the `Release` object without going through provider-kubernetes. So provider-kubernetes should never touch this, or am I missing something?

Reply:
yeah it seems you are right I will remove this.